.github/workflows/ci-pullrequest-main.yml

```diff
@@ -3,9 +3,14 @@ name: Mock Bot CI Pipeline
 on:
   pull_request:
     branches: [ main ]
+    
+env:
+  Bot1Url: "http://localhost:9012"
```

Comment:
Are these environment variables ever used? assume these are ones that need to be passed to the docker compose file?

Reply:
They are used here, but they are the same as app settings (these will override app settings), wanted them in here so that we can easily set different values if we need to. They will be when we use this after the CD stage and need to set real azure urls so wanted to have them here too